unit/EspmTest.cpp

```diff
@@ -379,3 +379,21 @@ TEST_CASE("Correctly parses tree structure", "[espm]")
             0x4f838, 0x50015, 0x69857, 0x6ed38, 0x94b35, 0xc350d, 0xc97eb,
             0xd45f0, 0x104217 });
 }
+
+TEST_CASE("Parsing RACE", "[espm]")
+{
+  auto& br = l.GetBrowser();
+
+  auto form = br.LookupById(
+    0x0001B1DB); // Ri'saad(Roving merchant from Khajiit's caravan.)
```

Comment:
Please move comment to a separate line, a short function argument on a separate line looks strange